src/Symfony/Component/Security/Http/Event/CredentialsVerificationFailedEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Symfony\Component\Security\Http\Event;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Http\Authenticator\AuthenticatorInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * This event is dispatched after an error during authentication.
+ *
+ * Listeners to this event can change state based on authentication
+ * failure (e.g. to implement login throttling).
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+class CredentialsVerificationFailedEvent extends Event
```

Comment:
I'm not sure this name represents what it does well - it's not really related to credentials failing - just authentication failing in general, right?

Reply:
Agreed, renamed to `LoginFailedEvent` & `LoginSuccessEvent`